subsys/net/ip/net_private.h

```diff
@@ -34,6 +34,9 @@ union net_mgmt_events {
 #if defined(CONFIG_NET_IPV6) && defined(CONFIG_NET_IPV6_MLD)
 	struct net_event_ipv6_route ipv6_route;
 #endif /* CONFIG_NET_IPV6 && CONFIG_NET_IPV6_MLD */
+#if defined(CONFIG_WIFI_FEAT_RAW_SCAN_RESULTS)
+	char raw_scan_result_event[CONFIG_WIFI_RAW_SCAN_RESULT_LENGTH];
```

Comment:
I don't think I understand:
* Why isn't `data` buffer in `struct wifi_raw_scan_result` set to `CONFIG_WIFI_RAW_SCAN_RESULT_LENGTH`
* And instead, raw buffer of this side is provided here instead of the actual event struct.